docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+*   ``conda install dask``               : Installs dask from the default channel (recommended)
+*   ``conda install -c conda-forge dask``: Installs dask from `conda-forge <https://conda-forge.github.io/>`_.
```

Comment:
I suspect that having both commands here will confuse new users, who will think that they need to run both commands. I'm inclined to stick with the "packages are maintained on both channels" message from before.

There is also some cost to adding a new channel here if the user doesn't use conda-forge (or more likely just doesn't understand the concept of channels).  The dependency packages like numpy are still slightly different on the two channels, so this will likely cause a significant download.  My experience is that people who know about conda-forge are also well-enough informed to add the `-c conda-forge` while users who don't know about conda-forge are generally not well served with the added choice.

Reply:
Agreed, will keep it simple as it is. I do like the format xarray uses (see [here](https://github.com/pydata/xarray/blob/6ccbdfd065e135bc58c409ff8fb6b2f4739fb4d5/doc/installing.rst#instructions)), but again, that's probably for advanced use cases only.